notebooks/108-gpu-device/108-gpu-device.ipynb

```diff
@@ -0,0 +1,928 @@
+{
```

Comment:
<p>I think it is good place also for link on AUTO device tutorial too</p><p>https://github.com/openvinotoolkit/openvino_notebooks/tree/main/notebooks/106-auto-device</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/788/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>